src/covid_shared/cli_tools/run_directory.py

```diff
@@ -88,19 +88,20 @@ def setup_directory_structure(output_root: Union[str, Path], with_production: bo
 
 
 def make_links(app_metadata: Metadata, run_directory: Path,
-               mark_as_best: bool, production_tag: Optional[str]) -> None:
+               mark_as_best: bool, production_tag: Optional[str], quick: int) -> None:
     if app_metadata['success']:
-        mark_latest(run_directory)
-
-        if mark_as_best:
-            mark_best(run_directory)
-
-            if production_tag:
-                try:
-                    mark_production(run_directory, production_tag)
-                except ValueError:
-                    logger.warning(f'Invalid production tag {production_tag}. Run not marked for production.'
-                                   f'Please provide production tag in format YYYY_MM_DD.')
+        if not quick:
```

Comment:
if app_metadata['success'] and not quick:

Reply:
Heh, that's embarrassing